fdm-app/app/components/custom/farm/farm-pagination.tsx

```
@@ -0,0 +1,45 @@
+
+export function FarmPagination({
+    items,
+    currentPath,
+    children,
+}: PaginationLayoutProps) {
+    if (!items?.length) return null
+
+    return (
+        <div className="space-y-6">
+            <Pagination aria-label="Farm navigation">
+                <PaginationContent role="navigation">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<nav>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)